### ISSUE TITLE
Way to filter out status codes?

### ISSUE THREAD

**Issue by jooj123** [elastic/apm-agent-nodejs#1123]
**Is your feature request related to a problem? Please describe.**
Would be great if it was possible to filter out certain status codes (eg: 404, 410 etc) with this apm client.

**Describe the solution you'd like**

A option in config option to specify an array of status codes to ignore listed here:
https://www.elastic.co/guide/en/apm/agent/nodejs/current/configuration.html

**Describe alternatives you've considered**
Can do it as part of a filter https://www.elastic.co/guide/en/apm/agent/nodejs/current/agent-api.html#apm-add-filter but this seems a bit annoying - a config property seems simpler

**Comment by coolboi567** [comment]
Have you considered the alternative to pass [`ignoreUrls`](https://www.elastic.co/guide/en/apm/agent/nodejs/master/configuration.html#ignore-urls) in the configuration options?

```
require('elastic-apm-node').start({
  ignoreUrls: [
    '/ping',
    /^\/admin\//i
  ]
})
```

If you know the URLs endpoints which causes are likely to get 404 spams then, I think using this option would suffice.

**Comment by watson** [comment]
If possible I'd recommend the URL filter that @coolboi567 suggested. With this, we save some resources, but not having to wait until the status code is received to know if we should collect it or not. Would that be an ok solution for you or are the URL patterns that gives the 4xx status codes not known to you in advance?

**Comment by jooj123** [comment]
The problem is we get a ton of bots - I think we are going to need a status code filter
`ignoreUrls` wont cut it for the volume we get, sometimes there is no pattern to the URLs at all

**Comment by watson** [comment]
Ok, I see. I'll bring it up with the team

**Comment by jooj123** [comment]
I guess i can use `ignoreUserAgents` to filter out bots .. 

but sometimes you want to still check if bots are running into requests that are actually problematic (eg 500+ errors) so you dont want to filter them out entirely

**Comment by watson** [comment]
yeah, I think the `addFilter` approach you suggested as a workaround is best. It allows you to do exactly what's needed. But having a dedicated config option would of course be easier.

**Comment by jooj123** [comment]
`addFilter`? Do youm mean `ignoreUserAgents` ?

**Comment by watson** [comment]
No I mean what you suggested in the issue description:

> **Describe alternatives you've considered**
> Can do it as part of a filter https://www.elastic.co/guide/en/apm/agent/nodejs/current/agent-api.html#apm-add-filter but this seems a bit annoying - a config property seems simpler
